style/android/java/install.bat

```diff
@@ -0,0 +1,22 @@
+REM Installs Thoughtbot's IntelliJ configs into your user configs.
+@echo off
+echo Installing Thoughtbot IntelliJ configs...
+
+setlocal enableDelayedExpansion
+
+for /D %%i in (%userprofile%\.AndroidStudio*) do call :copy_config %%i
+for /D %%i in (%userprofile%\.IdeaIC*) do call :copy_config %%i
+for /D %%i in (%userprofile%\.IntelliJIdea*) do call :copy_config %%i
+
+echo.
+echo Restart IntelliJ and/or AndroidStudio, go to preferences, and apply 'ThoughtbotAndroid'.

```

Comment:
As a member of the  "And/or"  Hater Club, I must request that you go with "or" instead.